chapters/equations.tex

```diff
@@ -696,6 +696,8 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 The equations of a \lstinline!when!-clause are active during initialization, if and only if they are explicitly enabled with \lstinline!initial()!, and only in one of the two forms \lstinline!when initial() then! or \lstinline!when {$\ldots$, initial(), $\ldots$} then! (and similarly for \lstinline!elsewhen! and algorithms see below).
 In this case, the \lstinline!when!-clause equations remain active during the whole initialization phase.
 In case of a \lstinline!reinit(x, expr)! being active during initialization (due to being inside \lstinline!when initial()!) this is interpreted as adding \lstinline!x = expr! (the \lstinline!reinit!-equation) as an initial equation.
+The \lstinline!reinit! handling applies both if directly inside \lstinline!when!-clause or inside an \lstinline!if!-equation in the \lstinline!when!-clause.
+If \lstinline!reinit(x, expr)! is inside an \lstinline!if!-equation in the \lstinline!when!-clause it is treated as and counted as the equation \lstinline!x = expr;! for the \lstinline!if!-equations according to \cref{if-equations}.
```

Comment:
Maybe we could formulate this as a consequence of the previous sentence instead of something that looks like an independent statement?
```suggestion
In particular, a \lstinline!reinit(x, expr)! needs to be counted as the equation \lstinline!x = expr;! for the purpose of balancing of \lstinline!if!-equations inside \lstinline!when!-clauses that are active during initialization, see \cref{if-equations}.
```